server_extensions_extra/src/main/resources/batm-extensions.xml

```diff
@@ -1967,5 +1968,17 @@
         <cryptologo cryptocurrency="ILC" file="ilc.png"/>
         <cryptologo cryptocurrency="ILC" file="ilc.svg"/>
     </extension>
+    <extension class="com.generalbytes.batm.server.extensions.extra.cardano.CardanoExtension" >
+        <wallet prefix="cardano" name="Cardano">
+            <param name="protocol" />
+            <param name="host" />
+            <param name="port" />
+            <param name="wallet_id" />
+            <param name="wallet_passphrase" />
+            <cryptocurrency>ADA</cryptocurrency>
+        </wallet>
```

Comment:
Add <help>help text here. See how other extensions use help tag in this to describe service or parameters.</help>